openslides/motions/static/js/motions/site.js

```diff
@@ -806,6 +806,26 @@ angular.module('OpenSlidesApp.motions.site', [
         });
         $scope.alert = {};
 
+        // collect all states and all recommendations of all workflows
+        $scope.states = [];
+        $scope.recommendations = [];
+        var workflows = Workflow.getAll();
+        _.forEach(workflows, function (workflow) {
+            if (workflows.length > 1) {
```

Comment:
This check is always true because if workflows is an empty array the loop won't run.